python/ray/autoscaler/aws/cloudwatch/cloudwatch_waiter_for_prometheus_metrics.sh

```diff
@@ -0,0 +1,15 @@
+MAX_ATTEMPTS=120
+DELAY_SECONDS=10
+RAY_PROM_METRICS_FILE_PATH="/tmp/ray/prom_metrics_service_discovery.json"
+CLUSTER_NAME=$1
+while [ $MAX_ATTEMPTS -gt 0 ]; do
+  if [ -f $RAY_PROM_METRICS_FILE_PATH ]; then
+    echo "Restarting cloudwatch agent.This may take a few minutes..."
```

Comment:
echo "Ray Prometheus metrics service discovery file found at: $RAY_PROM_METRICS_FILE_PATH."
echo "Restarting cloudwatch agent. This may take a few minutes..."